### PR TITLE
ACM-5402: Fixes problem when synchronizing informers

### PR DESCRIPTION
**Pull request by jlpadilla**
### Related Issue
https://issues.redhat.com/browse/ACM-5402

### Description of changes
- The loop to poll CRDs and start the informers was blocked on `initialized <- struct{}{}`
- Refactored the code to write to the initialized channel only once.
- Minor log adjustments.

### CODE REVIEW

File: pkg/informer/supportedResources.go [hunk 1]
```diff
@@ -62,30 +62,30 @@ func isResourceAllowed(group, kind string, allowedList []Resource, deniedList []
...
 			return true
 		}
 	}
-	glog.V(2).Infof("Deny resource [group: '%s' kind: %s]. It doesn't match any allow or deny rule.", group, kind)
+	glog.V(3).Infof("Deny resource [group: '%s' kind: %s]. It doesn't match any allow or deny rule.", group, kind)
 	return false
 }
 
-//helper funtion to get map of resource object
+// Helper funtion to get map of resource object
```
Comment (SherinV) [inline pkg/informer/supportedResources.go:88]:
```suggestion
// Helper function to get map of resource object
```

Comment (jlpadilla) [reply pkg/informer/supportedResources.go]: done

**Review by SherinV — APPROVED** [review]

**Comment by jlpadilla** [comment]
/override ci/prow/sonar-pre-submit
This PR improves test coverage, but doesn't meet the gate. Overriding gate to allow continuous improvement in this legacy project.

**Pull request merged**